werkzeug/wrappers.py

```diff
@@ -1072,15 +1083,11 @@ def set_cookie(self, key, value='', max_age=None, expires=None,
                          extension to the cookie standard and probably not
                          supported by all browsers.
         """
-        self.headers.add('Set-Cookie', dump_cookie(key,
-                                                   value=value,
```

Comment:
Can we keep the vertical alignment of all keys? IMO easier to read. Of course the linebreak before the first one is an improvement.